tests/core/test_crawler_rpc.py

```diff
@@ -1,26 +1,12 @@
-import atexit
```

Comment:
This is covered in https://github.com/Chia-Network/chia-blockchain/pull/10614.

Reply:
This is on top of #10614. Let me see if I can separate them.